.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig http://EditorConfig.org
+
+# Project Root
+root = true
+
+# Default Code Style
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+tab_width = 4
```

Comment:
I'm saying `tab_width = 8` is a sensible default because it's the de facto standard and it aligns with what the python interpreter expects.  If you set it to 4 here we'll need to set it to 8 in the [*.py] block.

I don't see a reason to set this to 4 for our code base at all, so we should either remove this line or set it to 8 so it doesn't conflict with python's interpretation of tabs.